### PR TITLE
Allow using the same version as the platform

### PR DESCRIPTION
**Pull request by mikeurbanski1**
# In This PR

- Accepts the value `platform` for the version, in which case it will use the platform API to get the checkov version during installation.

## Pictures/videos

- [X] I've reviewed my own code

### CODE REVIEW

File: src/configuration.ts [hunk 1]
```diff
@@ -46,14 +46,27 @@ export const getUseDebugLogs = (): boolean | undefined => {
...
     const checkovVersion = configuration.get<string>('checkovVersion', 'latest').trim().toLowerCase();
 
     if (checkovVersion === '' || checkovVersion === 'latest') {
         return 'latest';
+    } else if (checkovVersion === 'platform') {
+        logger.debug('Checkov version is "platform" - getting version from platform API');
+
+        const token = configuration.get<string>('token');
+        const prismaApiUrl = configuration.get<string>('prismaURL');
+        if (!token) {
```
Comment (kartikp10) [inline src/configuration.ts:61]: The checkovVersion endpoint does not require authentication. I think this step can be skipped

File: src/utils.ts [hunk 2]
```diff
@@ -219,6 +222,46 @@ const parseRepoName = (repoUrl: string): string | null => {
...
 export const getTokenType = (token: string): TokenType => token.includes('::') ? 'prisma' : 'bc-token';
 
+export const getPlatformCheckovVersion = async (token: string, prismaApiUrl: string | undefined, logger: winston.Logger): Promise<string> => {
+    
+    const url = `${prismaApiUrl ? `${prismaApiUrl}/code/` : 'https://www.bridgecrew.cloud/'}${checkovVersionApi}`;
+
+    logger.debug(`Checkov version URL: ${url}`);
+
+    if (prismaApiUrl) {
+        token = await getPrismaLoginToken(token, prismaApiUrl);
```
Comment (kartikp10) [inline src/utils.ts:232]: This can be removed as well

**Review by kartikp10 — APPROVED** [review]
Looks good! Left a minor comment. How do you feel about using radio buttons for version selection?
[x] platform
[ ] latest
[ ] ________

**Comment by mikeurbanski1** [comment]
I do not think it's possible to have a radio button with a free text field like that. All we can do is specify metadata in package.json and VSCode builds the settings page.

**Review by try-panwiac — APPROVED** [review]

**Pull request merged**